src/path.ml

```diff
@@ -26,3 +26,8 @@ let relative_all p xs = List.fold_left Filename.concat p xs
 let join x y = Filename.concat x y
 
 let withExt x ~ext = x ^ ext
+
+let split x =
+  x |> Js.String.split sep |> Array.to_list |> List.filter (fun p -> p != "")
+
+let concat x = sep ^ (x |> Array.of_list |> Js.Array.joinWith sep)
```

Comment:
Can't you just use `String.concat`? Also, the name `Path.concat` seems confusing. The resultant path is going to be absolute, which is definitely not something I'd expect.

Reply:
Sorry, I made this PR quickly, just to explore the idea, so code and naming may be not optimized